File: scripts/services/ntt_global_network.py
from core.statuspage import fetch_statuspage

SERVICE = {
    "id": 'ntt-global-network',
    "name": 'NTT Global Network',
    "desc": '全球 Tier-1 IP 骨幹與跨洋網路',
    "category": 'backbone',
    "page_url": 'https://status.act.ntt.com',
    "data_url": 'https://status.act.ntt.com/api/v2/incidents.json',
}

def fetch(ctx):
    return fetch_statuspage(SERVICE, ctx)

File: assets/services/ntt-global-network.js
/* CloudStatus service module: NTT Global Network */
(function () {
  "use strict";
  window.CloudStatusServices.register({
  "id": "ntt-global-network",
  "name": "NTT Global Network（NTT 全球骨幹網）",
  "desc": "全球 Tier-1 IP 骨幹與跨洋網路",
  "category": "backbone",
  "page": "https://www.nttdata.com/global/en/services/connectivity/global-ip-network",
  "parser": "ntt-global",
  "sources": [
    {
      "type": "reader",
      "label": "官方頁",
      "tier": 50,
      "kind": "official-status",
      "priority": 50
    }
  ]
});
})();
